src/main/java/com/epam/esm/model/User.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.esm.model;
+
+import org.hibernate.envers.Audited;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "Users")
+@Audited
+public class User implements BaseModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "Login")
+    private String login;
+
+    @Column(name = "Password")
+    private String password;
+
+    @OneToMany(mappedBy = "user",fetch = FetchType.EAGER,
```

Comment:
Do we really need fetchtype=eager? If we need info about user why should we fetch orders too.
Also add additional info about user, e.g. name, birth date.

Reply:
When we get info about users, we get user params (login, password and etc.) and user orders. That's why we need fetchtype=eager.

Reply:
Now user have name and age params